src/core/validators/validators.ts

```diff
@@ -144,9 +158,9 @@ const Url: ValidatorFn = (value) => {
   return Promise.resolve(null);
 };
 
-const isEmptyValue = (value: Value) => {
-  return  _.isEmpty(value) || (typeof value==='string' && value?.trim?.() === '');
-}
+const isEmptyValue = (value: Value) =>
+  (_.isEmpty(value) && !_.isDate(value)) ||
+  (_.isString(value) && value.trim() === '');
```

Comment:
Would be nice to have a generic solution to handle others types, I am thinking about numbers for instance

Reply:
_.isEmpty(value) || _.isNill(value)) ||
  (_.isString(value) && value.trim() === '')
Something like this could work?

Reply:
Ok, after some advanced research, I propose the following


```
return ((_.isArray(value) || _.isPlainObject(value)) && _.isEmpty(value)) || _.isNil(value) || (_.isString(value) && value.trim() === '');
```

Reply:
Yes, thank you :)!!  This was the idea. The `_.isEmpty` needed to be combined with other method to not give false positives.